plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -189,41 +169,13 @@ public boolean performOk() {
     return false;
   }
 
-  private void savePreferences() throws IOException {
-    IPreferenceStore preferenceStore = getPreferenceStore();
-
-    preferenceStore.setValue(PREF_PROJECT_ID, model.getProjectId());
-    preferenceStore.setValue(PREF_PROMPT_FOR_PROJECT_ID, model.isPromptForProjectId());
-    preferenceStore.setValue(PREF_OVERRIDE_DEFAULT_VERSIONING, model.isOverrideDefaultVersioning());
-    preferenceStore.setValue(PREF_CUSTOM_VERSION, model.getVersion());
-    preferenceStore.setValue(PREF_ENABLE_AUTO_PROMOTE, model.isAutoPromote());
-    preferenceStore.setValue(PREF_OVERRIDE_DEFAULT_BUCKET, model.isOverrideDefaultBucket());
-    preferenceStore.setValue(PREF_CUSTOM_BUCKET, model.getBucket());
-
-    ScopedPreferenceStore scopedPreferenceStore = (ScopedPreferenceStore) preferenceStore;
-    if (scopedPreferenceStore.needsSaving()) {
-      scopedPreferenceStore.save();
-    }
+  private void savePreferences() throws BackingStoreException {
+    model.savePreferences();
```

Comment:
Would it make sense to move the exception logging into `savePreferences()` and instead have `savePreferences()` return true/false on success?